javascripts/background_scripts/context_menu.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @fileOverview This file creates context menus.
+ * Clicking certain context menu item will start the
+ * app in seamless-posting mode.
+ */
+/*global chrome */
+var createSeamlessMenuItem = function (itemCaption, appName) {
+  chrome.contextMenus.create({
+    title: itemCaption,
+    contexts: ['editable'],
+    onclick: function (info, tab) {
+      // We use native message sending method here
+      // since we want to send to a specific target.
+      // context_messenger doesn't support specify
+      // target now.
+      chrome.tabs.sendRequest(tab.id, {
```

Comment:
It looks like this message interface is [deprecated](https://developer.chrome.com/extensions/tabs#method-sendRequest)